edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/appsettings_agent.json

```diff
@@ -25,5 +25,6 @@
   "EdgeModuleVolumeName": "",
   "EdgeHubVolumePath": "",
   "EdgeModuleVolumePath": "",
-  "ConfigRefreshFrequencySecs": "3600"
+  "ConfigRefreshFrequencySecs": "3600",
+  "WorkloadStaleSocketExceptionCode": 111
```

Comment:
Where is the value 111 coming from? 
How do we know iotedged will throw this error code?